guides/source/configuring.md

```diff
@@ -455,11 +455,11 @@ to `'http authentication'`.
 * `config.action_dispatch.signed_cookie_salt` sets the signed cookies salt value.
 Defaults to `'signed cookie'`.
 
-* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt
-value. Defaults to `'encrypted cookie'`.
+* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
 
-* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed
-encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
+* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
+
+* `config.action_dispatch.authenticated_encrypted_cookie_salt` sets the authenticated encrypted cookie salt. Defaults to `'authenticated encrypted cookie'`.
```

Comment:
Agree the middleware is out of scope.

I'm fine with making it configurable. And then we'll need to mention it in the security, but stress that it's not something that most apps should use:

```
Rails provides a default salt for the authenticated encryption. In the rare case you want to change that use `config.action_dispatch.authenticated_encrypted_cookie_salt`.
```